src/IISIntegration/dependencies.overrides.props

```diff
@@ -4,9 +4,9 @@
   </PropertyGroup>
   <PropertyGroup Label="Package Versions: Pinned">
   <!-- The AspNetCoreModule package versions are pinned to an older version for forwards and backwards compatibility tests. -->
-    <MicrosoftAspNetCoreAspNetCoreModuleStablePackageVersion>2.2.0-preview3-35497</MicrosoftAspNetCoreAspNetCoreModuleStablePackageVersion>
```

Comment:
revert this.

Reply:
Don't we want 2.2.1 to test against 2.2.0 RTM?

Reply:
We can, but it would require a few test updates. A few tests will fail in the test run for forwards and backwards compat because the check a hard coded assembly version.

Reply:
This will also fix our issue of needed to handle myget stuff.

Reply:
Good point. I'll revert this then and let you take care of updating tests separately. This change isn't necessary to get the 2.2.1 patch rolling.